doc/GETSTARTED.md

```diff
@@ -0,0 +1,77 @@
+# Get Started
+
+- Skill requirements
+
+  - Operating system
+
+    We will need a unix-based operating system to run Elektra (Linux, BSD, macOS) but it's also possible to use Windows which is supported but not tested much. If you are new to the world of unix we recommend you to use one of the common linux distributions like [Ubuntu](https://ubuntu.com/download/desktop#download), [Linux Mint](https://www.linuxmint.com/download.php), [openSuse](https://software.opensuse.org/distributions/leap) or [Fedora](https://getfedora.org/en/).
+
+  - Using command-line interface and commands
+
+    The easiest way to compile, install and use Elektra is by using the terminal. We will introduce the basic commands which you will need to run Elektra for the very first time.
+
+  - Basic knowledge about git
+
+    Dont panic! [GIT](https://git-scm.com/) is a distributed version constrol system to track changes of the source code in a project. We will use a single command of GIT to get
```

Comment:
I like this sort of writing but I think that those sentences here are not needed. If somebody wants to contribute to a repository on github he will for sure know about git. Just link git website in the "Basic knowledge about git"

Reply:
I also like the style. Maybe some students really do not know git, so I think it is okay to have it (maybe once in the whole tutorial)